src/test/java/emily/task/TaskTest.java

```diff
@@ -0,0 +1,14 @@
+package emily.task;
+
+import main.java.emily.task.Task;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I think it would be good to avoid wildcard imports? I noticed this in a few other places as well.